dask/array/slicing.py

```diff
@@ -1240,3 +1240,38 @@ def cached_cumsum(seq, initial_zero=False):
     if not initial_zero:
         result = result[1:]
     return result
+
+
+def take_along_axis(arr,indices,axis=None):
+    out_d = np.array([]) #creating a new empty array to store the value of result
+    arr = np.array(arr)
+    indices = np.array(indices)
+    if arr.shape != indices.ndim:
+        raise ValueError(
+            "`indices` and `arr` must have the same number of dimensions")
+    for integer in indices:
+        if not isinstance(integer,int):
+            raiseIndexError('indices must be an integer array')
+    if axis is None:
+        arr = arr.flatten
+        arr_shape = (np.prod((arr.shape),)
```

Comment:
This line has a mismatch in the number of open, `(`, and close, `)`, parenthesis. That's what is leading to the `SyntaxError` in the CI builds

Reply:
Thank you, I will look into it.